include/freerdp/redirection.h

```diff
@@ -43,6 +43,9 @@
 
 #define LB_PASSWORD_MAX_LENGTH 512
 
+#define ELEMENT_TYPE_CERTIFICATE 32
```

Comment:
Curious, why did you make these public definitions?

Reply:
Because when the certificate is set with `redirection_set_byte_option` the certificate is in `TARGET_CERTIFICATE_CONTAINER` format, which uses these two values, and to store it, these values are checked in `rdp_redirection_read_target_cert`.

So making these values public helps build the certificate externally in `TARGET_CERTIFICATE_CONTAINER` format and keep compatibility.

If we set the certificate in plain DER format, this would be simpler and these values wouldn't need to be public.